soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -251,85 +251,85 @@ void InitTrickNames() {
      Text{"Magical Shield", "Bouclier Magique", "Escudo arcano"},
      Text{"Mirror of Twilight", "Miroir des Ombres", "Espejo del Crepúsculo"}};
   trickNameTable[GI_TUNIC_GORON] = {
-     Text{"Gerudo Tunic", "Tunique Gerudo", "Sayo gerudo"},
-     Text{"Magic Armor", "Armure Magique", "Túnica Goron"},
+     Text{"Gerudo Top", "Corsage de damoiselle", "Pechera gerudo"},
+     Text{"Flamebreaker Armor", "Armure de pierre", "	Armadura ignífuga"},
      Text{"Red Mail", "Habits Rouges", "Ropas rojas"}};
   trickNameTable[GI_TUNIC_ZORA] = {
-     Text{"Rito Tunic", "Tunique Rito", "Sayo rito"},
+     Text{"Mermaid Suit", "Costume de sirène", "Costume de sirène"},
```

Comment:
mermaid suit is an oracle of ages refernce https://zeldawiki.wiki/wiki/Mermaid_Suit

Reply:
Indeed, but I feel like the original Rito stuff was more fitting on the wording and all ^-^

Reply:
Yeah, not sure why it needs to be removed in cases like this. You could have just added to the list instead.

Reply:
I'm going to push a commit that expands the lists and includes all the old ones that aren't obvious problems, so as not to hold up real issues because of subjective things.